docs/source/conf.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is mostly boilerplate from `sphinx-quickstart` but I made a few tweaks to customize the docs page.